NDcPP_v4_0-draft.adoc

```diff
@@ -844,15 +844,15 @@ Loss of communication with the audit server is problematic. While there are seve
 * _Changes to TSF data related to configuration changes (in addition to the information that a change occurred it shall be logged what has been changed)._
 * _Generating/import of, changing, or deleting of cryptographic keys (in addition to the action itself a unique key name or key reference shall be logged)._
 * _[selection: Resetting passwords (name of related Administrator account shall be logged), no other actions, [assignment: list of other uses of privileges]];_
-. _Specifically defined auditable events listed in_ _Table 2_.
+. _Specifically defined auditable events listed in_ _Table 2_ and applicable auditable events defined in the claimed PP-Module(s) and functional package(s).
```

Comment:
I think we've lost some consistency in the way auditable events are specified since they can now come from additional places. This is one of 5 suggested changes in FAU_GEN.1.1 and Application Notes 1, 3, 4, and 48. Similar to X.509, I worked to consolidate things into Application Note 4 to reduce redundancy and ease maintenance.

Since this SFR will be copied into the ST, I believe the Application Notes should specify what is included in the auditable events table, the table will be customized to the TOE, I don't think the modules and packages should be mentioned in the SFR.
```suggestion
. _Specifically defined auditable events listed in_ _Table 2_.
```